### PR TITLE
Feat10/test setup

### PR DESCRIPTION
**Pull request by joelgrand**
Working on #10:
* defines a test project (Test) using MSTest-framework
* first unit tests created
* first integration tests created

### CODE REVIEW

**Review by guentgia — CHANGES_REQUESTED** [review]
Great job Gioelo! Thank you very much. The test environment with MS test looks good. As we just discussed, please have a look at the comments for requested changes.

File: D-lama-service/Test/IntegrationTests/UserControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        /// 
+        /// API PATH: /api/User
+        /// METHOD: POST
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        [TestMethod]
+        public async Task CreateUserTest() 
+        {
+            // # 1. Invalid request object.
```
[the remark this answers is not in the document]
Comment (joelgrand) [reply D-lama-service/Test/IntegrationTests/UserControllerTest.cs]: done now

File: D-lama-service/Test/IntegrationTests/UserControllerTest.cs [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MSTest.IntegrationTests
```
[the remark this answers is not in the document]
Comment (joelgrand) [reply D-lama-service/Test/IntegrationTests/UserControllerTest.cs:9]: renamed them

File: D-lama-service/Test/IntegrationTests/UserControllerTest.cs [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+        {
+            var uri = _apiRoute + "/AuthToken";
+            var content = new StringContent(JsonConvert.SerializeObject(login), Encoding.UTF8, "application/json");
+            var request = new HttpRequestMessage(HttpMethod.Post, uri);
+            request.Content = content;
+            var response = await _client.SendAsync(request);
+            return response.Content.ReadAsStringAsync().Result;
+        }
+
+
```
[the remark this answers is not in the document]
Comment (joelgrand) [reply D-lama-service/Test/IntegrationTests/UserControllerTest.cs]: done as well

**Review by guentgia — COMMENTED** [review]
Thank you @joelgrand ! Looks great.

**Review by guentgia — APPROVED** [review]

**Pull request merged**